.eslintrc

```diff
@@ -31,7 +31,7 @@
     "react/jsx-no-constructed-context-values": 0,
     "react/prop-types": 0,
     "no-shadow": "off",
-    "no-console": 0,
+    "no-console": "warn",
```

Comment:
Not a good idea in my oppinion. If for development purpose yu need console you can disable it for specific line/file